toolkit/tools/precacher/precacher.go

```diff
@@ -59,6 +59,7 @@ var (
 	outDir            = app.Flag("output-dir", "Directory to download packages into.").Required().ExistingDir()
 	snapshot          = app.Flag("snapshot", "Path to the rpm snapshot .json file.").ExistingFile()
 	outputSummaryFile = app.Flag("output-summary-file", "Path to save the summary of packages downloaded").String()
+	repoSummaryFile   = app.Flag("repo-summary-file", "Path to save the list of packages available in the repos").String()
```

Comment:
Can you rephrase to clarify this is the list of URLs to the packages, not just package names themselves?